src/test/java/net/datafaker/providers/base/PhoneNumberTest.java

```diff
@@ -28,14 +34,12 @@ void testCellPhone_enUS() {
         assertThat(cellPhone).matches("\\(?\\d+\\)?([- .]\\d+){1,3}");
     }
 
-    @Test
+    @RepeatedTest(COUNT)
     void testPhone_esMx() {
         final BaseFaker f = new BaseFaker(new Locale("es", "MX"));
         final PhoneNumber phoneNumber = f.phoneNumber();
-        for (int i = 0; i < 10; i++) {
-            assertThat(phoneNumber.cellPhone()).matches("(044 )?\\(?\\d+\\)?([- .]\\d+){1,3}");
-            assertThat(phoneNumber.phoneNumber()).matches("\\(?\\d+\\)?([- .]\\d+){1,3}");
-        }
+        assertThat(phoneNumber.cellPhone()).matches("(044 )?\\(?\\d+\\)?([- .]\\d+){1,3}");
+        assertThat(phoneNumber.phoneNumber()).matches("\\(?\\d+\\)?([- .]\\d+){1,3}");
```

Comment:
The groupings may as well be non-capturing where possible.

Reply:
@kingthorin I agree, but now I feel out of resources to refactor these old regexp as well. :(

Reply:
No problem